tests/bookmarks_and_history/test_open_all_bookmarks_from_bookmarks_toolbar.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+from selenium.webdriver import Firefox
+
+from modules.browser_object_context_menu import ContextMenu
+from modules.browser_object_navigation import Navigation
+from modules.browser_object_panel_ui import PanelUi
+from modules.page_object_generics import GenericPage
+
+
+@pytest.fixture()
+def test_case():
+    return "2084564"
+
+
+URL1_TO_BOOKMARK = "https://www.reddit.com/"
+URL2_TO_BOOKMARK = "https://www.youtube.com/"
+URL3_TO_BOOKMARK = "https://www.mozilla.org/"
+
+
+def test_open_all_bookmarks_from_bookmarks_toolbar(driver: Firefox):
+    """
+    C2084564: Verify that the user can Open all the bookmarks from the Bookmarks Toolbar
+    """
+    # instantiate object
+    nav = Navigation(driver)
+    panel = PanelUi(driver)
+    context_menu = ContextMenu(driver)
+    page = GenericPage(driver)
+
+    # Have a few Bookmarks saved on the Toolbar
+    urls_to_bookmark = [URL1_TO_BOOKMARK, URL2_TO_BOOKMARK, URL3_TO_BOOKMARK]
+    for url in urls_to_bookmark:
+        GenericPage(driver, url=url).open()
+        nav.add_bookmark_via_star()
+
```

Comment:
After this loop, we should load a page that we didn't bookmark, so we can ensure that we're not just picking up on that instance of the page.

Reply:
I added 2 lines to load a page that we didn't bookmark and to open bookmarks toolbar. Thanks